features/cli/vulnerability_show.feature

```diff
@@ -0,0 +1,100 @@
+Feature: CLI vulnerability show command
+
+  Scenario Outline: CLI vulnerability show
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I verify that running `pro vulnerability show invalid` `as non-root` exits `1`
+    Then I will see the following on stderr:
+      """
+      Error: issue "invalid" is not recognized.
+      Usage: "pro vulnerability show CVE-yyyy-nnnn" or "pro vulnerability show USN-nnnn"
+      """
+    When I push static file `security_issues_xenial` to machine
+    And I run `pro vulnerability show CVE-2024-7542 --data-file=/tmp/security_issues_xenial` as non-root
+    Then I will see the following on stdout:
+      """
+      CVE-2024-7542 does not affect your system
+      """
+    When I run `apt update` with sudo
+    And I run `apt install vim=2:7.4.1689-3ubuntu1.5 -y` with sudo
+    And I run `pro vulnerability show <cve> --data-file=/tmp/security_issues_xenial` as non-root
+    And I remove colors from output
+    Then stdout matches regexp:
+      """
+      CVE-2022-2286
```

Comment:
Could we pick a CVE which has `[NOTES]` on it? This way we can test all the blocks. See the comment in the code, there is a slight mismatch in formatting there, which we can capture here.